package.json

```diff
@@ -39,6 +43,13 @@
         }
       ]
     },
+    "viewsWelcome": [
+      {
+        "view": "FileTreeView",
+        "contents": "Open folder where model files are stored. Models files are one of the following format:\n- *.tflite or saved model for Tensorflow​\n- *.onnx or *.pth for PyTorch\n[Open Folder](Command:onevscode.fileTreeView-open)\nNext, register already-installed toolchain or install a new toolchain. You can do this later.\n[Register Installed Toolchain](Command:onevscode.register-installed-toolchain)\nor\n[Install New Toolchain](Command:onevscode.install-new-toolchain)\n",
```

Comment:
`.pb` is omitted intentionally?

Reply:
I will add. Thanks. 
@llFreetimell and @mhs4670go, please let me know if ONE supports more extensions.

Reply:
@hyunsik-yoon Sorry for bothering you :( `keras_model` is also supported...

Reply:
Thanks. I added Keras model into the description string.